### ISSUE TITLE
From Console : E/TbsReaderView: not supported by:doc ? 

### ISSUE THREAD

**Issue by HassanDev13** [hanlyjiang/AndroidDocumentViewer#7]
did he suport doc file ?

**Comment by hanlyjiang** [comment]
Yes, but it must be successfully initialized, although successful initialization may depend on the network

**Comment by hanlyjiang** [comment]
> did he suport doc file ?

Yes, but it must be successfully initialized, although successful initialization may depend on the network
